.travis.yml

```diff
@@ -100,7 +100,7 @@ before_script:
   - |
     for meta in "${WHEEL_METAS[@]}"; do
         _f=${meta#wheels/} ; wheel=${_f%%/*}
-        wheel_type=$($STARFORGE_CMD wheel_type --wheels-config="$meta" "$wheel")
+        wheel_type=$($STARFORGE_CMD wheel_type --wheels-config="$meta" "$wheel") || return $?
```

Comment:
I don't understand why [this build](https://travis-ci.org/galaxyproject/starforge-recipes/jobs/424332534) is not failing here.

Reply:
Swapping `return` with `exit` made the builds fail.